crates/client/eth/src/l1_messaging.rs

```diff
@@ -53,14 +56,13 @@ pub async fn sync(backend: &MadaraBackend, client: &EthereumClient, chain_id: &C
         }
     };
     let event_filter = client.l1_core_contract.event_filter::<StarknetCoreContract::LogMessageToL2>();
+
     let mut event_stream = event_filter
         .from_block(last_synced_event_block.block_number)
-        .select(BlockNumberOrTag::Finalized)
```

Comment:
we should add this back?

Reply:
it was causing the issue actually, was not able to get the messages